docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -93,12 +93,14 @@ done
 
 ```json
 {
-  "id": "fed-node",
-  "kind": "Minion",
-  "apiVersion": "v1beta1",
-  "labels": {
-    "name": "fed-node-label"
-  }
+    "apiVersion": "v1beta3",
+    "kind": "Node",
+    "metadata": {
+        "name": "fed-node"
+    },
+    "spec": {
+        "externalID": "fed-node"
```

Comment:
If it is not provided, it will be defaulted to name, which is what you are doing here anyway.